### ISSUE TITLE
General: Rethink Contact us link placement or menu option

### ISSUE THREAD

**Issue by wrendo** [cagov/design-system#475]
Let the debate begin

**Comment by adam-little** [comment]
I believe this is a duplicate of the issue here: https://github.com/cagov/design-system/issues/428

I have made a recommendation and have not heard any objections.

TLDR:

- Contact Link goes in the Site Footer since it is unique to the website.
- Do not include any contact link in Statewide Footer to avoid possible duplication and confusion
- For the _Design System Website specifically_, I recommend we put "Contact Us" in the Site Header *if* collecting feedback and connecting with potential users/collaborators is a key objective
- Note that this does not violate any accessibility rules as was previously thought

**Comment by chachasikes** [comment]
Tagging myself on this issue to implement it.

Comments:
- Move link to Site footer - sounds good
- I think I remember in our understanding/assumption is that Contact us is optional in the Statewide footer because we are providing a link to contact us in the overall footer area

- Where I left off in understanding the accessibility rule: For duplicate contact us links we decided we would try to get more information about what the exact accessibility rule from CDT really is - if it's the label or the URL, or both that causes an accessibility problem. @rkojik  will know or be able to find us the answer

In the meantime, I will move the Contact us link to the Site footer area.

**Comment by adam-little** [comment]
**Re: accessibility issue:** Please see [#428](https://github.com/cagov/design-system/issues/428). I had Greg Duncan confirm that there is no accessibility issue as long as we have the same label ("Contact Us") go to the same link (.../contact-us/).

**Comment by m-sullivan7** [comment]
>     * For the _Design System Website specifically_, I recommend we put "Contact Us" in the Site Header _if_ collecting feedback and connecting with potential users/collaborators is a key objective

I think by "site header", we mean the Design System [site navigation](https://designsystem.webstandards.ca.gov/components/menu/readme/), correct?

Also, thanks for the duplicate flag Adam!

**Comment by chachasikes** [comment]
This was assigned to me. Should have been assigned to Michael or Adam.
Is a dupe & dup issue is closed. Closing.
